config/core/build-configs-chromeos.yaml

```diff
@@ -57,6 +57,11 @@ build_configs:
     branch: 'linux-5.10.y'
     variants: *chromeos-variants
 
+  chromeos-stable_5.15:
```

Comment:
I guess we probably should also rename `chromeos-stable` to `chromeos-stable_5.10`

Reply:
Done